mappings/servers/minelandnetwork/metadata.json

```diff
@@ -0,0 +1,51 @@
+{
+    "id": "minelandnetwork",
+    "name": "Mineland Network",
+    "description": "Welcome to Mineland Network",
+    "primaryAddress": "mineland.net",

```

Comment:
Primary address should be the subdomain you included earlier (mc.)

And then you need to have this one in the addresses array

You clearly haven't read the docs...